RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitMapProducer.cc

```diff
@@ -0,0 +1,72 @@
+#ifndef __RecoLocalCalo_HGCRecProducers_HGCalRecHitMapProducer_H__
+#define __RecoLocalCalo_HGCRecProducers_HGCalRecHitMapProducer_H__
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
```

Comment:
this could be a global EDProducer, because no member variables are modified during produce()